processing/src/main/java/org/apache/druid/segment/data/GenericIndexed.java

```diff
@@ -530,13 +530,13 @@ public void close()
         headerOut.writeInt(Ints.checkedCast(valuesOut.size()));
 
         if (prevVal instanceof Closeable) {
-          CloseQuietly.close((Closeable) prevVal);
+          CloseableUtils.closeAndWrapExceptions((Closeable) prevVal);
```

Comment:
This changes the behavior, but any callers of `fromIterableVersionOne()` doesn't seem creating `GenericIndexed` from `Closeable` types. How about removing this?

Reply:
If it's ok with you, I'd prefer to leave it in for this patch, because I didn't want to go through and make sure it's always ok to skip closing.